contracts/ToucanRegenBridge.sol

```diff
@@ -90,14 +86,10 @@ contract ToucanRegenBridge is Ownable, Pausable, AccessControl {
     //      Constructor
     // ----------------------------------------
 
-    constructor(address tokenIssuer_, INCTPool nctPool_) Ownable() {
-        tokenIssuer = tokenIssuer_;
+    constructor(INCTPool nctPool_) {
         nctPool = nctPool_;
+        _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
```

Comment:
Since we are going to assign these roles to other accounts as soon as the contract is deployed, we may as well provide the correct accounts as part of the constructor inputs.

Reply:
2 arrays, one with addresses and one with roles + a loop over them to assign roles. That's the design I'm thinking of.

Reply:
I actually feel more comfortable keeping this in its own commit, I'm not sure in which other commit it would make sense to add it

Reply:
We should avoid granting access to `msg.sender` in case my original comment wasn't clear and only assign to the provided accounts.